modules/installation-azure-confidential-vms.adoc

```diff
@@ -0,0 +1,91 @@
+// Module included in the following assemblies:
+//
+// * installing/installing_azure/installing-azure-network-customizations
+
+:_content-type: PROCEDURE
+[id="installation-azure-confidential-vms_{context}"]
+= Enabling Confidential VMs
+
+:FeatureName: Confidential VM
```

Comment:
This should go right in front of the tech preview snippet inclusion

It also doesn't read very well. Should it be something like "Using confidential VMs"?